test/indexer/test_hca_indexer.py

```diff
@@ -626,6 +626,8 @@ def test_imaging_bundle(self):
         for hit in hits:
             entity_type, aggregate = config.parse_es_index_name(hit['_index'])
             sources[entity_type, aggregate].append(hit['_source'])
+            assay_type_count = 1 if aggregate else 240  # bundle has 240 items in its imaging_protocol['target']
```

Comment:
What's the point again of maintaining a large number of identical strings in the ES document?

Reply:
'assay_type' in `_protocol_dict()` is a list of values which allows duplicates to allow the  `FrequencySetAccumulator` to accumulate and return a list of the highest frequency items.

Reply:
See above.